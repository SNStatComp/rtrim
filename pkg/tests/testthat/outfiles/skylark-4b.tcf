FILE outfiles/skylark_wt.dat
TITLE  skylark-4b
NTIMES 8
NCOVARS 2
LABELS
Habitat
Cov2
End
MISSING 999
WEIGHT present

comment Now with preset changepoints
model 2
changepoints 1 2
weighting on
overdisp off
serialcor off
run
