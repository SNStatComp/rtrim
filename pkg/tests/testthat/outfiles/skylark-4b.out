 TRIM 3.70 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 3-10-2016 11:53:53

 Title :   skylark-4b

 Comment: Now with preset changepoints

 The following  6 variables have been read from file: 
 Z:\Users\patrick\Work\CBS\TRIM\wine\skylark_wt.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. weight 
 5. HABITAT          number of values:  2
 6. COV2             number of values:  4

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
                                              Weighted  Weighted
   TimePoint  Observations  Average   Index   Average   Index  
       1              25       8.52    1.00     30.12    1.00
       2              20       8.10    0.95     27.00    0.90
       3              30      10.90    1.28     24.10    0.80
       4              30      11.27    1.32     23.87    0.79
       5              28      12.75    1.50     23.36    0.78
       6              29      14.66    1.72     36.38    1.21
       7              22      17.00    2.00     31.73    1.05
       8              18      18.89    2.22     35.39    1.17

 RESULTS FOR MODEL: Linear Trend
 --------------------------------
  Changes in Slope at Timepoints
      1     2 

 WEIGHTING = On

 ESTIMATION METHOD = Maximum Likelihood


  Total time used:  0.72 seconds

 GOODNESS OF FIT
  Chi-square                190.45, df    145, p 0.0000
  Likelihood Ratio          186.75, df    145, p 0.0000
  AIC (up to a constant)   -103.25

 WALD-TEST FOR SIGNIFICANCE OF CHANGES IN SLOPE 
 Changepoint    Wald-Test  df   p  
      1            13.44    1  0.0002
      2            18.63    1  0.0000

 PARAMETER ESTIMATES 

 Slope for Time Intervals
 from upto   Additive  std.err.    Multiplicative   std.err.
    1    2   -0.3108    0.0848          0.7329       0.0621
    2    8    0.0798    0.0120          1.0831       0.0130

 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7329     0.0621       0.7530     0.0721
     3     0.7938     0.0639       0.7947     0.0739
     4     0.8597     0.0669       0.8109     0.0748
     5     0.9311     0.0716       0.8059     0.0762
     6     1.0084     0.0786       0.9241     0.0860
     7     1.0922     0.0881       0.9706     0.0898
     8     1.1829     0.1007       1.0027     0.0984

 TIME TOTALS
  Time     Model     std.err.    Imputed     std.err.
     1      1424         119        1533         127
     2      1044          68        1154          83
     3      1130          67        1218          85
     4      1224          67        1243          88
     5      1326          70        1235          93
     6      1436          77        1417         101
     7      1556          88        1488         106
     8      1685         105        1537         121

 OVERALL SLOPE MODEL (with intercept)
  Additive      std.err.   Multiplicative  std.err.
    0.0472       0.0103         1.0484      0.0108

 OVERALL SLOPE IMPUTED (with intercept, recommended)
  Additive      std.err.   Multiplicative  std.err.
    0.0206       0.0114         1.0209      0.0116

 OVERALL SLOPE MODEL (through base time point)
  Additive      std.err.   Multiplicative  std.err.
    0.0017       0.0156         1.0017      0.0156

 OVERALL SLOPE IMPUTED (through base time point)
  Additive      std.err.   Multiplicative  std.err.
   -0.0199       0.0163         0.9803      0.0160

